test/provision/k8s_install.sh

```
@@ -178,7 +178,7 @@ case $K8S_VERSION in
         KUBERNETES_CNI_VERSION="0.6.0"
         K8S_FULL_VERSION="1.13.0-beta.2"
         KUBEADM_OPTIONS="--ignore-preflight-errors=cri"
-        KUBEADM_SLAVE_OPTIONS="--discovery-token-unsafe-skip-ca-verification --ignore-preflight-errors=cri"
+        KUBEADM_SLAVE_OPTIONS="--discovery-token-unsafe-skip-ca-verification --ignore-preflight-errors=cri,SystemVerification"
```

Not blocking, just out of curiosity - what is the need to ignore this error?